src/common/utils/deployUtils.ts

```diff
@@ -1140,7 +1140,13 @@ export async function buildOrgManifest(
         parsedPackageXml.Package.types.push(newType);
       }
     }
-
+    // Sort members for each type once after the loop
+    for (const type of parsedPackageXml.Package.types) {
+      if (type.members && Array.isArray(type.members)) {
```

Comment:
if you want to even improve perfs, you can sort only the types that are in mdTypes :)